react/src/reactTextMask.js

```diff
@@ -52,12 +37,12 @@ export const MaskedInput = React.createClass({
     return (
       <input
         {...props}
-        onInput={this.onChange}
+        onInput={event => this.onChange(event)}
```

Comment:
@zackify Why not put the arrow on the function declaration or bind in constructor?
I see no reason to instantiate a new function at each render.